tests/resources/codeflare-stack/codeflare-subscription.yaml

```diff
@@ -8,3 +8,5 @@ spec:
   name: codeflare-operator
   source: community-operators
   sourceNamespace: openshift-marketplace
+  installPlanApproval: Manual
```

Comment:
I didn't test this but I wonder if one of the operators is using Manual, and the other operators are waiting for this codeflare-operator?